app/views/devise/sessions/new.html.erb

```diff
@@ -8,6 +8,14 @@
 <% end %>
 <%= render 'shared/maintenance_window_alert' %>
 
+<% if decorated_session.sp_name && decorated_session.irs_attempts_api_enabled %>
+  <%= render AlertComponent.new(
+        type: :info,
+        class: 'margin-bottom-2',
+        message: t('notices.irs_data_use.text_html', link: link_to(t('notices.irs_data_use.link'), MarketingSite.security_and_privacy_practices_url)),
+      ) %>
```

Comment:
If we take the `link` variable out, we can do this
```suggestion
  <%= render AlertComponent.new(type: :info, class: 'margin-bottom-2') do %>
    <%= t('notices.irs_data_use.text_html') %>
    <%= link_to(t('notices.irs_data_use.link'), MarketingSite.security_and_privacy_practices_url) %>
  <% end %>
```

Reply:
This is no longer relevant